adabot/circuitpython_bundle.py

```diff
@@ -321,18 +337,6 @@ def update_bundle(bundle_path):
             )
         )
         release_required = True
-    if update_download_stats(bundle_path):
```

Comment:
I also removed this code that updates the pypi stats. We're no longer using the stats in the file used for meeting notes so this does not need to update any longer.

Reply:
If we don't use this, we can probably remove the Google BigQuery task that occurs daily that ultimately feeds into this.  I don't think we're billed over it, but probably worth doing if we're not using the download stats :) It's attached to my account I believe so I can do that.